dashboard/dashboard_builder.py

```diff
@@ -378,7 +367,8 @@ def main():
         process.expect('Ready for new connection', timeout=5)
 
         packages = [
-            package.Package(install_name) for install_name in args.packages]
+            package.Package(install_name) for install_name in args.packages
+        ]
```

Comment:
It's interesting that the brackets are buffered by new lines in this list comprehension definition.